ProcessMaker/bpm
# Responsive Columns

fixes #1082 
To test:
set screen to 320 + and start a request.
all pages should now have responsive breakpoints :) 
Data tables have a horizontal scroll for overflow 
tabs become stacked
![screen shot 2018-12-21 at 10 50 55 am](https://user-images.githubusercontent.com/29641725/50358807-72ca0000-050f-11e9-9e80-e7ea28f5135b.png)
![screen shot 2018-12-21 at 10 51 18 am](https://user-images.githubusercontent.com/29641725/50358808-72ca0000-050f-11e9-84e4-b8d0b671da67.png)
![screen shot 2018-12-21 at 10 51 32 am](https://user-images.githubusercontent.com/29641725/50358809-72ca0000-050f-11e9-99dd-ed30ff3abb57.png)
